### ISSUE TITLE
_document.attachEvent is not a function - NativeScript Angular Issues

### ISSUE THREAD

**Issue by sean-perkins** [danielgek/nativescript-sentry#7]
When trying to implement this plugin in a NativeScript Angular environment, underlying web-related functions are being called. 

Resulting error trace:
```
CONSOLE LOG file:///app/core/services/ns-app.service.js:42:16: toggling http logs: OFF
CONSOLE ERROR file:///app/tns_modules/tns-core-modules/trace/trace.js:165:30: ns-renderer: ERROR BOOTSTRAPPING ANGULAR
CONSOLE ERROR file:///app/tns_modules/tns-core-modules/trace/trace.js:165:30: ns-renderer: _document.attachEvent is not a function. (In '_document.attachEvent('onclick', self._breadcrumbEventHandler('click'))', '_document.attachEvent' is undefined)
CONSOLE WARN file:///app/tns_modules/tns-core-modules/ui/text-view/text-view.js:14:46: Class "UITextViewDelegateImpl" already implements the "UIScrollViewDelegate" protocol.
11:03:21 AM - Compilation complete. Watching for file changes.
```

Minimum steps to reproduce:
1. Add module import on the `AppModule`
```
SentryModule.forRoot({
     dsn: 'https://foo:foo2@com.domain.example/10'
}),
```

(we have a valid dsn, just not wanting to share the public one).

On the latest of everything:
```
All NativeScript components versions information
┌──────────────────┬─────────────────┬────────────────┬─────────────┐
│ Component        │ Current version │ Latest version │ Information │
│ nativescript     │ 3.4.0           │ 3.4.0          │ Up to date  │
│ tns-core-modules │ 3.4.0           │ 3.4.0          │ Up to date  │
│ tns-android      │ 3.4.0           │ 3.4.0          │ Up to date  │
│ tns-ios          │ 3.4.0           │ 3.4.0          │ Up to date  │
└──────────────────┴─────────────────┴────────────────┴─────────────┘
```

Any ideas?

**Comment by danielgek** [comment]
@sean-perkins just published 1.5.1 can you try it ?

**Comment by piotrilski** [comment]
@danielgek I can confirm that with 1.5.1 the issue is gone.

**Comment by sean-perkins** [comment]
I can confirm that with 1.5.1 this issue is **not** gone:
```
CONSOLE ERROR file:///app/tns_modules/raven-js/src/console.js:32:36: Unhandled Promise rejection: _document.createEventObject is not a function. (In '_document.createEventObject()', '_document.createEventObject' is undefined) ; Zone: <root> ; Task: Promise.then ; Value: TypeError: _document.createEventObject is not a function. (In '_document.createEventObject()', '_document.createEventObject' is undefined) _triggerEvent@file:///app/tns_modules/raven-js/src/raven.js:809:40
CONSOLE ERROR file:///app/tns_modules/raven-js/src/console.js:32:36: Error: Uncaught (in promise): TypeError: _document.createEventObject is not a function. (In '_document.createEventObject()', '_document.createEventObject' is undefined)
11:05:18 AM - Compilation complete. Watching for file changes.
```

package-lock bit:
```
{
  "_from": "nativescript-sentry@^1.5.1",
  "_id": "nativescript-sentry@1.5.1",
  "_inBundle": false,
  "_integrity": "sha1-P+LOzmj6w5CIHENzWIdTlwdvSKM=",
  "_location": "/nativescript-sentry",
  "_phantomChildren": {},
  "_requested": {
    "type": "range",
    "registry": true,
    "raw": "nativescript-sentry@^1.5.1",
    "name": "nativescript-sentry",
    "escapedName": "nativescript-sentry",
    "rawSpec": "^1.5.1",
    "saveSpec": null,
    "fetchSpec": "^1.5.1"
  },
  "_requiredBy": [
    "/"
  ],
  "_resolved": "https://registry.npmjs.org/nativescript-sentry/-/nativescript-sentry-1.5.1.tgz",
```

**Comment by danielgek** [comment]
have to investigate more

**Comment by danielgek** [comment]
@sean-perkins released 1.5.2, but unsure if its fixed

**Comment by Burgov** [comment]
@danielgek doesn't look like it. Installed 1.5.2 and I'm getting the same error

**Comment by Burgov** [comment]
Fixed it by putting 

```

import Raven = require("raven-js");

(<any>Raven)._hasDocument = false
```

in my app.module.ts file, but ideally this wouldn't be necessary

**Comment by bozhidarc** [comment]
I'm getting the following error now.
I'm with latest version of the plugin 1.5.2.
```
TypeError: _document.createEventObject is not a function. (In '_document.createEventObject()', '_document.createEventObject' is undefined)
```

**Comment by alexandruantonica** [comment]
Hi guys :) 
The same here.

**Comment by danielgek** [comment]
this will fix it self once https://github.com/NativeScript/nativescript-angular/pull/1410 gets released

**Comment by mpochron** [comment]
This is fixed after using nativescript-angular@6.1.0 :) 
CONFIRMED.
